samples/ballerina-by-example/examples/strings/strings.sh

```diff
@@ -0,0 +1,26 @@
+# To run the program, create a file named `strings.bal`
+# and run the command below.
+
+$ $BALLERINA_HOME/bin/ballerina strings.bal
+ToUpper: LION IN TOWN. CATCH THE LION
+ToLower: lion in town. catch the lion
+EqualsIgnoreCase: true
+SubString: Lion
+Contains: true
+IndexOf: 2
+LastIndexOf: 26
+ValueOf: 5.8
+ReplaceFirst: Tiger in Town. Catch the Lion
+Replace: Tiger in Town. Catch the Tiger
+ReplaceAll: Li0n in T0wn. Catch the Li0n
+Length: 29
+Trim: Lion in Town. Catch the Lion
+HasSuffix: true
+HasPreffix: true
+Unescape: Lion in Town. Catch the Lion
+Split: Lion
+Split: in
+Split: Town.
+Blob: Lion in Town. Catch the Lion
+
+# Next example:
```

Comment:
Remove this line since the linking to next sample happens though examples.txt file.